src/_posts/2022-05-17-restructuring-openoakland.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Restructuring OpenOakland"
+author: OpenOakland
+layout: post
+date: 2022-05-17 00:00:00 -0800
+permalink: updates/:title/
+feat-img: /assets/images/blog/2022-05-07-special-vote-results.png
+feat-img-alt: "Illustration of a voting button in primary colors."
+post-excerpt: "We're happy to report that membership unanimously voted yes to empower the Steering Committee to enter Lean Ops mode and focus on restructuring the brigade for greater inclusiveness. Our Lean Ops (lean operations) mode is intended to strip operational tasks down to the very barebones needed to accomplish two core goals..."
+---
+
+!["Illustration of a voting button in primary colors."](/assets/images/blog/2022-05-07-special-vote-results.png)
+
+In the absence of elected member representatives, OpenOakland recently held an open vote to determine whether or not membership wanted to empower the existing Steering Committee (currently made up of active project leads) to implement a Lean Ops mode in an effort to restructure OpenOakland to improve participation and more diverse representation.
+
+Eligible voters included anyone in good standing who attended an OpenOakland event or contributed to a project in the past year (with the exception of current Steering Committee members). **We're happy to report that fully 100% of eligible voting members voted _yes_ to empower the Steering Committee to enter Lean Ops mode and focus on restructuring the brigade for greater inclusiveness.**
+
+Although this vote may *seem* like a formality, this process helps us maintain accountability, ensure we're adhering to the spirit of our bylaws, and maintain paths to participation for volunteers interested in reshaping OpenOakland.
+
+## How much effort does it really take to run OpenOakland?
+
+There is a surprising amount of labor that goes into running the brigade. Onboarding new volunteers and matching them to appropriate opportunities requires one-on-one attention. Scheduling, structuring, and facilitating meetups every week requires hours of prep before the event even starts. Helping to manage and access accounts, documentation, and subscription services for multiple projects requires organization and communication with different people on different schedules. Communicating out to the public about brigade and project team activities requires hours of writing and publishing. And then there's the continuous effort of supporting member requests, building relationships with community partners, navigating interpersonal relationships, etc. Most of this is *good work* that helps us all have more of an impact. But it can be challenging for volunteers to do it all on top of our day jobs and personal lives.
```

Comment:
I feel so torn about this paragraph. It's 100% true and well-crafted to boot... but I just think it's wide of the mark. Arent' we going into Lean Ops mode because 1) the number of actively contributing volunteers has fallen off a cliff and therefore there is too much labor for those who remain; 2) we want to rebuild better. That means we're doing the slow, hard work of making sure past mistakes aren't repeated, and **while we do that work**, Lean Ops will carry us forward until we're ready to restructure.

Reply:
Yeah, I think maybe my original version wasn't super clear but that's essentially what I was trying to say. I made some edits to this section, particularly adding some of your language, which I think is really clear and important. Does it help?